docs/source/migration-guides/0.15-deserialization.md

```diff
@@ -0,0 +1,292 @@
+# Adjusting code to changes in deserialization API introduced in 0.15
+
+In 0.15, a new deserialization API has been introduced. The new API improves type safety and performance of the old one, so it is highly recommended to switch to it. However, deserialization is an area of the API that users frequently interact with: deserialization traits appear in generic code and custom implementations have been written. In order to make migration easier, the driver still offers the old API, which - while opt-in - can be very easily switched to after version upgrade. Furthermore, a number of facilities have been introduced which help migrate the user code to the new API piece-by-piece.
+
+The old API and migration facilities will be removed in the next major release (0.16).
+
```

Comment:
Please change this to "The old API and migration facilities will be removed in the future major release."

If our plans change and we make a new release without removing this (as it happened with serialization) it is too easy to have outdated docs.

I've forwarded you email from @annastuchlik that discourages such practices and provides examples from Scylla docs of such issues.